readme.md

```diff
@@ -1,5 +1,8 @@
 # Validate Date
 
+Simple date validation function. It takes a string in the shape of "mm/dd/yyyy" or "yyyy-mm-dd" and returns either a string with detailed information or a boolean value. Includes checking for leap years.
+
```

Comment:
Give me a string and I'll tell you if it's a valid date.

This package exports a single synchronous function that takes **date** as a “string” and/or a **response type** as a "boolean"(default=false) as input/s and returns a convenient response based on the user’s choice. It also validates leap years.

Reply:
I love your new phrasing!  I vote for keeping it. There were new changes after I added the other functionality. I will re-commit the readme.md with this phrasing and try to add the new part as similar as this

Reply:
And I agree with booleanResponse. It will definitely avoid headaches in the future.